lte/gateway/release/build-magma.sh

```diff
@@ -117,7 +117,7 @@ MAGMA_DEPS=(
     "lighttpd >= 1.4.45"
     "libxslt1.1"
     "nghttp2-proxy >= 1.18.1"
-    "python3-protobuf >= 3.14.0"
+    "python3-protobuf >= 3.19.0"
```

Comment:
Is there a reason for pinning it to exactly `==` 3.19.0 below, but allowing higher versions here?

Reply:
I agree, I think the versions in both setups should be the same.